timeline_data.json

```diff
@@ -1,7 +1,23 @@
 {
   "items": [
     {
-      "classes": ["featured-event"],
+      "date_str": "2023-06-07",
+      "title": "June 2023 Data Dump Not Posted",
+      "slug": "data-dump-not-posted",
+      "type": "licensing",
+      "summary": "Stack Exchange Inc. did not post the June 2023 data dump.",
+      "body": "<p>The job that uploads the data dump to Archive.org was disabled on 28 March, and marked to not be re-enabled without approval of senior leadership. This was mainly over concerns on AI using Stack Exchange data. This caused a large outrage and was eventually reverted on June 16, 2023 (9 days later).</p>",
```

Comment:
I'd go for a separate event for reinstatement with some additional information on how it transpired and remove the last sentence from here as a result.

Reply:
Happy to merge as-is, until another PR comes in adding a resolution event, or further details about this event.